packages/dashboard/src/layout/useLayoutManager.ts

```diff
@@ -5,7 +5,7 @@ import LayoutContext from './LayoutContext';
 /**
  * Retrieve the current layout from the context
```

Comment:
Update the comment to layout manager

Also update the context to `LayoutManagerContext` to match